manage.py

```diff
@@ -1,9 +1,45 @@
-#!/usr/bin/env python
```

Comment:
Don't remove this. It's unrelated to coverage and is necessary for running the script as `./manage.py`.

Reply:
Marking this as resolved.